src/utils/get-uk-date.js

```diff
@@ -1,6 +1,8 @@
 var DATA_NAME_CAMEL = 'noteEditedBy';
 var DATA_DATE_CAMEL = 'noteEditedDate';
 
+var config = require('../config');
+
 module.exports = function getUKDate(data){
```

Comment:
What about the Australians/Americans? :) Though I do like avoiding the moment dependency.